### PR TITLE
OSRA-348 Country names in Arabic

### PR DESCRIPTION
**Pull request by NikitaAvvakumov**
## Do not merge yet, waiting for client to confirm Arabic translations

https://osraav.atlassian.net/browse/OSRA-348

 - create config/locales/ar.yml with mappings of country alpha2 codes to full names in Arabic
 - in sponsor index & show views, display country names in Arabic with English in parentheses after
 - test display of English & Arabic country names in Cucumber scenario

While not directly requested by the client, this approach will allow us to demonstrate Rails' internationalization capabilities and to assure the client that separately storing data in Arabic is not necessary. Also, display of country names in both languages may help the client to catch any errors in translations before we start using the Arabic versions in orphan cards.

Complete lookup table can be seen at https://www.dropbox.com/s/n4fzgwzj7nn3o3z/CountryList%20complete%20en-ar.xlsx?dl=0

### CODE REVIEW

File: app/admin/sponsor.rb [hunk 1]
```diff
@@ -28,8 +28,8 @@
     column :start_date
     column :request_fulfilled
     column :sponsor_type
-    column :country do |sponsor|
-      ISO3166::Country.search(sponsor.country)
+    column :country do |_sponsor|
+      "#{t _sponsor.country, locale: :ar} (#{ISO3166::Country[_sponsor.country]})"
```
Comment (theB264) [inline app/admin/sponsor.rb]: With Arabic a RTL and English a LTR language, should this be formatted `(United Kingdom) المملكة المتحدة`

Comment (NikitaAvvakumov) [inline app/admin/sponsor.rb]: Valid point, but both you and I are guessing. I'm OK to leave it up to the client.

Comment (PurityControl) [inline app/admin/sponsor.rb]: as long as they realise they have the option.

**Comment by NikitaAvvakumov** [comment]
Country now displayed as `(English) Arabic`

**Comment by theB264** [comment]
:shipit:

**Comment by PurityControl** [comment]
Were still waiting for confirmation - right?

**Comment by NikitaAvvakumov** [comment]
Still waiting. But considering that we only really care about 4-5 countries, we can probably merge without and let them catch any mistakes after delivery.

**Comment by PurityControl** [comment]
do we want @motasem-salem to check the ones that matter before we merge ?

File: config/locales/ar.yml [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  TK:
+  TO: تونغا
+  TT: ترينيداد وتوباغو
+  TN: تونس
+  TR: تركيا
+  TM: تركمانستان
+  TV: توفالو
+  UG: أوغندا
+  UA: أوكرانيا
+  AE: الإمارات العرب
```
Comment (motasem-salem) [inline config/locales/ar.yml]: AE: الإمارات العربية المتحدة

**Comment by NikitaAvvakumov** [comment]
Updated with @motasem-salem's correction

**Pull request merged**